cob_moveit_bringup/launch/stomp_moveit_rviz.launch

```diff
@@ -0,0 +1,18 @@
+<launch>
```

Comment:
remove this file...you need to load the proper config locally...it's a duplicate anyway